pom.xml

```diff
@@ -137,6 +137,12 @@
             <version>${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
`junit-platform-suite` line 102-104 has `junit-platform-suite-api` as dependency, this one shouldn't be needed. Can you plz check if tests work without this dependency?